lib/honeybadger/insights/live_view.ex

```diff
@@ -0,0 +1,44 @@
+defmodule Honeybadger.Insights.LiveView do
+  @moduledoc false
+
+  use Honeybadger.Insights.Base
+
+  @required_dependencies [Phoenix.LiveView]
+
+  @telemetry_events [
+    "phoenix.live_component.handle_event.stop",
+    "phoenix.live_view.mount.stop",
+    "phoenix.live_view.update.stop"
+  ]
+
+  def extract_metadata(meta, _) do
+    %{
+      url: Map.get(meta, :uri),
+      socket_id: Map.get(meta, :socket_id),
+      view: extract_view(meta),
+      assigns: extract_assigns(meta),
+      params: Map.get(meta, :params),
+      event: Map.get(meta, :event)
+    }
+  end
+
+  defp extract_view(%{socket: socket}) do
+    socket.view |> get_module_name()
+  rescue
+    _ -> nil
+  end
+
+  defp extract_view(_), do: nil
+
+  defp extract_assigns(%{socket: socket}) do
+    socket.assigns
```

Comment:
Not sure if this is a concern or not, feel free to ignore.

When this data gets injected into the event, it's possible that it could include too much or unwanted data. Should we make this optional? Or is there a way to represent the data in another form?

Reply:
The data also gets run through the same filters as the params (I think of assigns as the params of liveview events), which is also why I updated the filtered params to remove some things from the assigns:

https://github.com/honeybadger-io/honeybadger-elixir/blob/4dc8bef4fc2aaae285d079989c6c2e1f76767900/mix.exs#L74